Examples/OneSignalDemo/app/src/androidTest/java/com/onesignal/sdktest/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.onesignal.sdktest;
```

Comment:
Can remove this file since we didn't create any tests for the example project.

Reply:
This is auto generated by Android Studio when making project, I can remove it